api/envoy/extensions/filters/listener/tls_inspector/v3/tls_inspector.proto

```diff
@@ -32,4 +32,9 @@ message TlsInspector {
   // tls inspector will consume.
   google.protobuf.UInt32Value initial_read_buffer_size = 2
       [(validate.rules).uint32 = {lt: 65537 gt: 255}];
+
+  // Populate ``JA4`` fingerprint hash using data from the TLS Client Hello packet.
+  // ``JA4`` is an improved version of ``JA3`` that includes TLS version, ciphers, extensions,
+  // and ALPN information in a hex format. Default is false.
```

Comment:
just to be clear, the default is both JA3 and JA4 fingerprinting are off?

Reply:
Yes, correct we want both to be off by default.